tests/test_wait_signal.py

```diff
@@ -335,7 +336,6 @@ class Obj(QtCore.QObject):
 
 
 class TestArgs:
-
```

Comment:
These line changes must have happened when I was using PyCharm's **Reformat code** functionality, which cleans up files very nicely. I'm not familiar with the general rule for this project. Normally, a project should define an open source tool that can be executed on a python module and it will format it according to rules set up by the project team. Is such a mechanism in place?